paperless/downloadfile.go

```diff
@@ -0,0 +1,80 @@
+package paperless
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var documentselection int
+
+func (p Paperless) setFullDownloadURL(location string) string {
+	switch p.UseHTTPS {
+	case true:
+		return "https://" + p.Hostname + location
+	case false:
+		return "http://" + p.Hostname + location
+	}
+	return ""
+}
+
+func (p Paperless) writeFile(document Document) {
+	downloadURL := p.setFullDownloadURL(document.DownloadURL)
+
+	client := http.Client{Timeout: time.Second * 5}
+	log.Debugf("downloading from: %v", downloadURL)
+	req, _ := http.NewRequest("GET", downloadURL, nil)
+	req.Header.Set("User-Agent", "paperless-cli")
+	req.SetBasicAuth(p.Username, p.Password)
```

Comment:
```suggestion
	req := ReturnAuthenticatedRequest(p.Username, p.Password)
```

`paperless/client.go` has a function to return an authenticated request for you:
You can do something like `req := ReturnAuthenticatedRequest(p.Username, p.Password)`

Reply:
👍